packages/frontend/web/server/htmlTemplate.ts

```diff
@@ -34,89 +32,75 @@ export const htmlTemplate = ({
             ? 'favicon-32x32.ico'
             : 'favicon-32x32-dev-yellow.ico';
 
+    const priorityScriptTags = priorityScripts.map(
+        src => `<script defer src="${src}"></script>`,
+    );
+
+    const lowPriorityScriptTags = lowPriorityScripts.map(
```

Comment:
We should make it clear that the `lowPriorityScriptTags` shouldn't ever rely on any of the `priortyScript`s (which is a little confusing) as there's no guarantees that the `deferred` will load and parse before `async`.

Reply:
Good point, I'll add a comment.